### PR TITLE
feat(live): add active and close chat for live component AP-538

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

Add active and close chat for live

**Which issue(s) this PR fixes** 🔨

AP-538

### CODE REVIEW

**Comment by WanderingHogan** [comment]
Can you update the component so it looks nicer with longer usernames?
Screen Shot 2022-02-02 at 12 57 04 PM

Also - when you hang up through the live component it doesn't appear to actually end the stream on the users side:
Screen Shot 2022-02-02 at 12 58 43 PM
Screen Shot 2022-02-02 at 12 58 34 PM

so when you reload now you still see the person you were in a call with's name

File: components/views/navigation/sidebar/live/Live.vue [hunk 1]
```diff
@@ -1,15 +1,55 @@
...
 
 export default Vue.extend({
   components: {
     RadioIcon,
     XIcon,
   },
+  computed: {
+    ...mapState(['ui', 'webrtc']),
+    existLiveChat(): boolean {
+      return this.$store.state.friends.all.find(
```
Comment (molimauro) [inline components/views/navigation/sidebar/live/Live.vue:16]: Use `some` if you want to return a boolean

**Comment by davidchan1219** [comment]
@WanderingHogan 
Screenshot_3
This problem isn't associated with live component. When click hangup button of media component, same issue is appeared.
I want you to create new ticket to solve this issue.

File: components/views/navigation/sidebar/live/Live.vue (hunk 1 above)
Comment (WanderingHogan) [reply components/views/navigation/sidebar/live/Live.vue]: I see he switched this to some, so marking as resolved for now

**Comment by WanderingHogan** [comment]
Ticket for webrtc disconnect added here: https://satellite-im.atlassian.net/browse/AP-691, otherwise works as expected, thank you

**Review by WanderingHogan — APPROVED** [review]

**Review by molimauro — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

File: components/views/navigation/sidebar/live/Live.vue [hunk 2]
```diff
@@ -1,15 +1,58 @@
...
     XIcon,
   },
+  computed: {
+    ...mapState(['ui', 'webrtc']),
+    existLiveChat(): boolean {
+      return this.$store.state.friends.all.some(
+        (friend) => friend.address === this.$store.state.webrtc.activeCall,
+      )
+    },
+    selUserName(): string {
```
Comment (stavares843) [inline components/views/navigation/sidebar/live/Live.vue:20]: I think this variable could have a better naming

File: components/views/navigation/sidebar/live/Live.vue [hunk 3]
```diff
@@ -1,15 +1,58 @@
...
+      const sUser = this.$store.state.friends.all.find(
```
Comment (stavares843) [inline components/views/navigation/sidebar/live/Live.vue:21]: I think this variable could have a better naming as well

**Pull request merged**